src/Findologic/Request/Handler/NavigationRequestHandler.php

```diff
@@ -151,6 +151,15 @@ private function fetchCategoryPath(Request $request, SalesChannelContext $salesC
             return null;
         }
 
-        return Utils::buildCategoryPath($category->getBreadcrumb());
+        if ($category->getId() === $salesChannelContext->getSalesChannel()->getNavigationCategoryId()) {
+            return null;
+        }
+
+        $navigationCategory = Utils::fetchNavigationCategoryFromSalesChannel(
```

Comment:
This is `$rootNavigationCategory` right? Does the helper method only return root categories? Maybe renaming method / variable helps.

Reply:
A Sales Channel only has one navigation category, therefore we're fetching it here, which results in this name.

Reply:
As discussed, renamed this to `$rootCategory`.